src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt

```diff
@@ -102,6 +107,29 @@ class HousekeepingEventListener : Listener, PacketListener {
             })
         }
 
+        event<PlayerInteractEvent> {
+            val player = this.player
+            val clickedBlock = clickedBlock ?: return@event
+
+            if (eventController().currentGame != null ||
```

Comment:
Invert this condition for readability: if current game is null OR clicked block is a snow variant, return.